core/examples/driver/main.cpp

```diff
@@ -75,12 +75,20 @@ int main(int /*argc*/, char** /*argv*/)
 {
```

Comment:
I saw this issue after posting the pull request.
I have fixed this issue and will submit the fix to this pull request.